src/services/fileStorage/proxy-service.js

```diff
@@ -33,9 +33,18 @@ const logger = require('../../logger');
 
 const FILE_PREVIEW_SERVICE_URI = process.env.FILE_PREVIEW_SERVICE_URI || 'http://localhost:3000/filepreview';
 const FILE_PREVIEW_CALLBACK_URI = process.env.FILE_PREVIEW_CALLBACK_URI
-|| 'http://localhost:3030/fileStorage/thumbnail/';
+	|| 'http://localhost:3030/fileStorage/thumbnail/';
 const ENABLE_THUMBNAIL_GENERATION = process.env.ENABLE_THUMBNAIL_GENERATION || false;
 
+const FILE_SECURITY_CHECK_SERVICE_URI = process.env.FILE_SECURITY_CHECK_SERVICE_URI
+	|| 'http://localhost:8081/scan/file';
+const FILE_SECURITY_CHECK_CALLBACK_URI = process.env.FILE_SECURITY_CHECK_CALLBACK_URI
+	|| 'http://localhost:3030/fileStorage/securityCheck/';
```

Comment:
dont we have the publivcServer API URL already available

Reply:
I'll check and update